jhipster/src/main/webapp/app/patient.tsx

```diff
@@ -9,90 +9,306 @@ import { useAppDispatch, useAppSelector } from 'app/config/store';
 
 import { getEntity } from './entities/patient/patient.reducer';
 
+import { getIMC } from './entities/imc/imc.reducer';
+import { getEpas } from './entities/epa/epa.reducer';
+import { getPoids } from './entities/poids/poids.reducer';
+import { Line } from 'react-chartjs-2';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+
+import '../content/css/patient.css';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend
+);
 export const Patient = () => {
   const dispatch = useAppDispatch();
 
   const { id } = useParams<'id'>();
 
   useEffect(() => {
     dispatch(getEntity(id));
+    dispatch(getIMC(id)).then((response) => {
```

Comment:
Do we really need to store IMC since we can calculate it with height and weight ?

Reply:
No,but we decided to make it an entity so we have to use it

Reply:
Ok